addons/project/views/project_project_views.xml

```diff
@@ -43,7 +43,7 @@
                     <div class="oe_button_box" name="button_box" groups="base.group_user">
                         <button class="oe_stat_button" type="object" name="action_view_tasks" icon="fa-check">
                             <div class="o_field_widget o_stat_info">
-                                <span class="o_stat_text">Tasks</span>
+                                <field name="label_tasks" readonly="1"/>
```

Comment:
```suggestion
                                <field name="label_tasks">
```

A related field is default readonly, so we don't need to specify the readonly attribute.